config/__init__.py

```diff
@@ -0,0 +1,26 @@
+# flake8: noqa
+from os import environ
+
+FLASK_ENV = environ.get("FLASK_ENV")
+print(FLASK_ENV)
+
+match FLASK_ENV:
+    case "tasks" | "development":
```

Comment:
is that meant to be "test"?

Reply:
No, it's for when we run stuff with invoke